main.go

```diff
@@ -95,6 +98,49 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func SetupLogging(opts *zap.Options) {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		log.Error(err, "Error getting config for APIServer")
+		os.Exit(1)
+	}
+
+	clnt, err := client.New(cfg, client.Options{})
+	if err != nil {
+		log.Error(err, "can't create client from config")
+		os.Exit(1)
+	}
+	oprNamespace, err := sdkk8sutil.GetOperatorNamespace()
+	if err != nil {
+		log.Error(err, "failed to get operator namespace")
+		os.Exit(1)
+	}
+
+	oprCfg, _ := configmaps.RetrieveConfigMap(oprNamespace, "artemiscloud-operator-config", clnt)
+	if oprCfg != nil && len(oprCfg.Data) > 0 {
+		if level, ok := oprCfg.Data["zap-log-level"]; ok {
```

Comment:
If we use a generic key like `log-level` it will work also if the `--zap-log-level` option will change

Reply:
yeah log-level seems better.